src/js/resize-manager.js

```diff
@@ -120,12 +131,8 @@ class ResizeManager extends Component {
       this.resizeObserver_.disconnect();
     }
 
-    if (this.el_ && this.el_.contentWindow) {
-      Events.off(this.el_.contentWindow, 'resize', this.debouncedHandler_);
-    }
-
-    if (this.loadListener_) {
```

Comment:
we should keep this around, while unlikely, it is still possible that dispose happens before the the loadListener is triggered.

Reply:
Or does this end up being cleaned up automatically? If so, maybe it doesn't need to be stored on `this` but just as a local variable?

Reply:
I think anything on `this` is cleaned up automatically, but I did not intend to remove this, let me fix it.